components/_shared/SideNavigation.vue

```diff
@@ -485,17 +511,32 @@ export default Vue.extend({
 
 .SideNavigation-Footer {
   padding-top: 20px;
-}
-
-.SideNavigation-Social {
   display: flex;
+
+  .SideNavigation-Footer-Rows {
+    flex-direction: row;
+
+    .SideNavigation-Social {
```

Comment:
```suggestion
    .SideNavigation-Social {
      display: flex;
```

`display:flex;` にすると何かと楽です。